src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $border-width;
+  padding: $padding;
+  width: $column-width;
```

Comment:
The variable `$column-width` is used here but not defined in this file. Ensure that it is defined somewhere in your SCSS files or imported from another file.